files/rustc/gcc.toml

```diff
@@ -0,0 +1,5 @@
+[[files]]
+name = "rustc/gcc/gmp-6.3.0.tar.bz2"
+source = "https://gmplib.org/download/gmp/gmp-6.3.0.tar.xz"
```

Comment:
`tar.xz` != `tar.bz2` and the checksum included here is of the `tar.xz` file, but gcc's `contrib/download_prerequisites` wants the `.tar.bz2`.